### PR TITLE
Update client setup API for Ecommerce site configurations

### PR DESCRIPTION
**Pull request by asadmanzoor93**
**Description:** This PR updates client setup API for Ecommerce site configurations

**JIRA:** 
https://edlyio.atlassian.net/browse/EDLY-3099

**Merge checklist:**

- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are (reasonably) squashed
- [ ] Translations are updated

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

**Note:** Please add screenshots for any viewable change.

### CODE REVIEW

File: ecommerce/extensions/edly_ecommerce_app/api/v1/views.py [hunk 1]
```diff
@@ -149,7 +150,7 @@ def process_client_sites_setup(self):
             lms_url_root=self.request.data.get('lms_site', '')
         )
         payments_site, __ = Site.objects.update_or_create(domain=payments_base, defaults=dict(name=payments_base))
-        payments_partner, __ = Partner.objects.update_or_create(short_code=edly_slug, default_site=payments_site)
+        payments_partner, __ = Partner.objects.update_or_create(short_code=edly_slug, default_site=payments_site, defaults=dict(name=edly_slug))
```
Comment (burhandodhy) [inline ecommerce/extensions/edly_ecommerce_app/api/v1/views.py:153]: `defaults=dict(name=edly_slug). -> defaults={'name': edly_slug}`

**Review by burhandodhy — COMMENTED** [review]
One minor change, rest is good

File: ecommerce/extensions/edly_ecommerce_app/api/v1/views.py (hunk 1 above)
Comment (asadmanzoor93) [reply ecommerce/extensions/edly_ecommerce_app/api/v1/views.py:153]: `defaults=dict(name=edly_slug)` is used in the code to keep consistency in code I have used a similar method.
Screenshot 2021-06-23 at 6 36 57 PM

**Review by zubair-arbi — APPROVED** [review]
LGTM 👍

**Review by burhandodhy — APPROVED** [review]

**Pull request merged**